README.md

```diff
@@ -230,6 +230,17 @@ mgit exec 'echo `pwd`'
 # /home/mgit/packages/organization/repository-2
 ```
 
+
+### dump
+
+Takes current hash commit from each cloned repository and saves them to the configuration file.
```

Comment:
Saves hashes of packages in `mgit.json`. It allows to easily fix project to a specific state.